src/System.Net.Http/src/System/Net/Http/Managed/ManagedHandler.cs

```diff
@@ -0,0 +1,307 @@
+            get { return _preAuthenticate; }
+            set { CheckInUse(); _preAuthenticate = value; }
+        }
+
+        public bool UseDefaultCredentials
+        {
+            get { return _useDefaultCredentials; }
+            set { CheckInUse(); _useDefaultCredentials = value; }
+        }
+
+        public ICredentials Credentials
+        {
+            get { return _credentials; }
+            set { CheckInUse(); _credentials = value; }
+        }
+
+        public bool AllowAutoRedirect
+        {
+            get { return _allowAutoRedirect; }
+            set { CheckInUse(); _allowAutoRedirect = value; }
+        }
+
+        public int MaxAutomaticRedirections
+        {
+            get { return _maxAutomaticRedirections; }
+            set { CheckInUse(); _maxAutomaticRedirections = value; }
+        }
+
+        public int MaxConnectionsPerServer
+        {
+            get { return _maxConnectionsPerServer; }
+            set
+            {
+                CheckInUse();
+                if (value <= 0)
+                {
+                    throw new ArgumentOutOfRangeException(nameof(value));
+                }
+
+                _maxConnectionsPerServer = value;
+            }
+        }
+
+        public int MaxResponseHeadersLength
+        {
+            get { return _maxResponseHeadersLength; }
+            set
+            {
+                CheckInUse();
+                if (value <= 0)
+                {
+                    throw new ArgumentOutOfRangeException(nameof(value));
+                }
+
+                _maxResponseHeadersLength = value;
+            }
+        }
+
+        public X509CertificateCollection ClientCertificates
+        {
+            get
+            {
+                if (_clientCertificateOptions != ClientCertificateOption.Manual)
+                {
+                    throw new InvalidOperationException(SR.Format(SR.net_http_invalid_enable_first, nameof(ClientCertificateOptions), nameof(ClientCertificateOption.Manual)));
+                }
+
+                return _clientCertificates ?? (_clientCertificates = new X509Certificate2Collection());
+            }
+        }
+
+        public Func<HttpRequestMessage, X509Certificate2, X509Chain, SslPolicyErrors, bool> ServerCertificateCustomValidationCallback
+        {
+            get { return _serverCertificateCustomValidationCallback; }
+            set { CheckInUse(); _serverCertificateCustomValidationCallback = value; }
+        }
+
+        public bool CheckCertificateRevocationList
+        {
+            get { return _checkCertificateRevocationList; }
+            set { CheckInUse(); _checkCertificateRevocationList = value; }
+        }
+
+        public SslProtocols SslProtocols
+        {
+            get { return _sslProtocols; }
+            set
+            {
+                CheckInUse();
+#pragma warning disable 0618 // obsolete warning
+                if ((value & (SslProtocols.Ssl2 | SslProtocols.Ssl3)) != 0)
```

Comment:
@morganbr, if SSL3.0 is not enabled by default and the developer needs to write specific code to use it, then they are choosing to use SSL3.0 including it's security limitations. If someone is communicating over a VPN, or the connection is only on a private secure network and never traverses an uncontrolled network, then using SSL3.0 isn't a problem. If you completely deny usage of SSL3.0, and it's the only option they have, then you are saying "If you have no other choice but to use SSL3.0, we don't want you using our framework, go use a different language and framework which lets you use SSL3.0".  
Are we really saying if you have no other options than to use SSL3.0 (one developer was in exactly that situation with hardware with an embedded web service they couldn't update/modify), then they should look at a different stack then .Net?